src/main/resources/com/puppycrawl/tools/checkstyle/messages_fr.properties

```diff
@@ -15,11 +15,11 @@ Main.errorCounter=Checkstyle se termine par {0} erreurs.
 Main.loadProperties=Impossible de charger des propriétés à partir du fichier ''{0}''.
 Main.createListener=Format de sortie invalide. Trouvé ''{0}'' mais attendu ''{1}'' ou ''{2}''.
 
-PackageObjectFactory.unableToInstantiateExceptionMessage = Impossible d'instancier la classe \
-  ''{0}'', il est également impossible de l''instancier comme {1}. \
+PackageObjectFactory.unableToInstantiateExceptionMessage = Impossible d`instancier la classe \
+  ''{0}'', il est également impossible de l``instancier comme {1}. \
```

Comment:
I think this should just be a single `.
The double `''` was because properties will convert it to a single `'` when displaying.